features/step_definitions/basic_steps.rb

```diff
@@ -30,3 +30,7 @@
 Then(/^show me the page$/) do
   save_and_open_page
 end
+
+Then(/^I should see "([^"]*)" link$/) do |link|
+  find_link(link).visible?
```

Comment:
You can do this or 
```ruby
expect(page).to have_selector('#selector', visible: true)
```
and `#selector` would be anything that matches the link
But I think the `find_link` method is straight to the point as it will look for a link that matches the text already.